### PR TITLE
Updated Dynamic filters

### PR DESCRIPTION
**Pull request by ThomasLandauer**
| Q | A
| --- | ---
| Branch? | 2.0
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Tests pass? | yes
| License | MIT
| Doc PR | yes

* Added info about autowiring. This is the most important part - took me some time to figure it out.
* "Moved" sample code from a controller to a service
* Deleted info about outdated Symfony 2.4

### CODE REVIEW

**Comment by maximgubar** [comment]
Hi @ThomasLandauer ,
Thank you for the merge request,
could you please merge latest master in your branch in order to pass the CI ?

**Comment by ThomasLandauer** [comment]
I got this mail, but it's not shown here:

> Hi @ThomasLandauer ,
Thank you for the merge request,
could you please merge latest master in your branch in order to pass the CI ?

Anyway, I don't know how to do that :-( I just changed the docs, so there shouldn't be a problem.

File: Resources/doc/filters.rst [hunk 1]
```diff
@@ -124,39 +124,67 @@ You can now reference and use your custom filter when defining filter sets in yo
...
+.. code-block:: yaml
+
+    # config/services.yaml
+    
+    services:
+        Liip\ImagineBundle\Imagine\Cache\CacheManager:
+            alias: liip_imagine.cache.manager
+        Liip\ImagineBundle\Imagine\Data\DataManager:
+            alias: liip_imagine.data.manager
+        Liip\ImagineBundle\Imagine\Filter\FilterManager:
```
Comment (dbu) [inline Resources/doc/filters.rst]: these alias are added in #1165 - i'd remove this part and only keep the service.

Comment (ThomasLandauer) [reply Resources/doc/filters.rst]: OK, great, I've updated it :-)

**Pull request merged**